odata-json-format/7 Structural Property.md

```diff
@@ -39,7 +39,9 @@ JSON strings whose content satisfies the rules `binaryValue`,
 `dateValue`, `dateTimeOffsetValue`,
 `durationValue`, `guidValue`, and
 `timeOfDayValue` respectively, in
-[OData-ABNF](#ODataABNF).
+[OData-ABNF](#ODataABNF). The interpretation of a `timeOfDayValue` in which the `second` is omitted
+is not defined by this specification. For maximum interoperability, clients and servers
```

Comment:
We avoid "server" and use "service" instead.

We could replace "clients and servers" with "senders"